examples/indexing.lua

```diff
@@ -4,42 +4,45 @@ local col = db:getCollection("exampleCO")
 
 function printResult(res)
 	for key in res do
-		for k,v in pairs(key) do
-			print(k,v)
+		for k, v in pairs(key) do
+			print(k, v)
 		end
 	end
 end
 
 local sample = {
-	{name = "orange", txt='Color between red and yellow', cost = 11},
-	{name = "grape", txt='Fruiting berry', cost = 3, lang='french'},
-	{name = "mango berry", txt='Juicy stone fruit', cost = 25},
-	{name = "grape", txt='Duplicate Fruiting berry', cost = 3},
-	{name = "banana", txt='Edible berry of flowering plants', cost = 1.5},
-	{name = "persimmon", txt='Edible fruit of trees', cost = 4.1}
+	{name = "orange", txt = 'Color between red and yellow', cost = 11},
+	{name = "grape", txt = 'Fruiting berry', cost = 3, lang='french'},
+	{name = "mango berry", txt = 'Juicy stone fruit', cost = 25},
+	{name = "grape", txt = 'Duplicate Fruiting berry', cost = 3},
+	{name = "banana", txt = 'Edible berry of flowering plants', cost = 1.5},
+	{name = "persimmon", txt = 'Edible fruit of trees', cost = 4.1}
 }
 print('\nInserting sample data:')
 col:insert_many(sample)
-printResult( col:find({}) )
+printResult(col:find({}))
 
-local res=col:createIndex( {name='text', txt='text'}, { 
-	name='myindex', 
-	default_language='english', 
-	language_override='lang', 
-	unique=false, 
-	sparse=true, 
-	background=true, 
-	expireAfterSeconds=100, 
-	textIndexVersion=1,
-	weights={name=10, txt=3} 
-} );
-print('\nCreating text search index: '..tostring(res))
+local indx = col:createIndex({name = 'text', txt = 'text'}, { 
+--	name = 'myindex', 
```

Comment:
I think I was attempting to demonstrate that you can a valid index name returned even if you do not provide one as a parameter. I agree that this is not necessary, so I will uncomment this line.